experimental/sdk/webpubsub-socketio-extension/examples/chat-with-negotiate/package.json

```diff
@@ -7,10 +7,11 @@
     "private": true,
     "license": "BSD",
     "dependencies": {
-      "express": "~4.17.1",
-      "@azure/web-pubsub-socket.io": "1.0.0-beta.2"
+        "@azure/identity": "^3.3.0",
+        "@azure/web-pubsub-socket.io": "1.0.0-beta.2",
```

Comment:
use beta.3?

Reply:
I think it would be updated to beta.5 later